src/config.rs

```diff
@@ -315,14 +316,19 @@ impl ConfigFile {
         self.source.path.as_ref().and_then(|path| path.parent())
     }
 
-    pub fn bytes(&self) -> &[u8] {
+    pub fn bytes(&self) -> &str {
         &self.bytes
     }
 
     fn resolve_pos(&self, pos: usize) -> LineCol {
-        let line = self.line_starts.iter().find(|&&start|
-            start < pos
-        ).copied().unwrap_or(self.line_starts.len());
+        let line = self.line_starts.iter().enumerate().find_map(|(i, start)|
+            if *start > pos {
+                Some(i)
+            }
+            else {
+                None
+            }
+        ).unwrap_or(self.line_starts.len());
         let line = line - 1;
         let col = self.line_starts[line] - pos;
```

Comment:
Not part of the diff, but, shouldn't this be the other way around, so `pos - self.line_starts[line]` ? `pos` is greater than the start of the line, and `col` should be the (positive) difference between them, no?

Reply:
Er, yes.